FBSimulatorControl/Commands/FBSimulatorMediaCommands.h

```diff
@@ -24,6 +24,29 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (FBFuture<NSNull *> *)addMedia:(NSArray<NSURL *> *)mediaFileURLs;
 
+/**
```

Comment:
These methods were public previously.
I think these can be public.

https://github.com/facebook/idb/commit/69ec3936bc0337a5a489b7d4f9a004c3860fd0ad#diff-33ff26a5ed6718867baaaeaa24f2cd8fL1